sdk/program/src/slot_history.rs

```diff
@@ -1,211 +1 @@
-        }
-        assert_eq!(slot_history.check(slot), Check::Found);
-    }
-
-    #[test]
-    fn slot_history_test_wrap() {
-        solana_logger::setup();
-        let mut slot_history = SlotHistory::default();
-        info!("add 2");
-        slot_history.add(2);
-        assert_eq!(slot_history.check(0), Check::Found);
-        assert_eq!(slot_history.check(1), Check::NotFound);
-        for i in 3..MAX_ENTRIES {
-            assert_eq!(slot_history.check(i), Check::Future);
-        }
-        info!("add 20");
-        slot_history.add(20);
-        info!("add max_entries + 19");
-        slot_history.add(MAX_ENTRIES + 19);
-        for i in 0..19 {
-            assert_eq!(slot_history.check(i), Check::TooOld);
-        }
-        assert_eq!(slot_history.check(MAX_ENTRIES), Check::NotFound);
-        assert_eq!(slot_history.check(20), Check::Found);
-        assert_eq!(slot_history.check(MAX_ENTRIES + 19), Check::Found);
-        assert_eq!(slot_history.check(20), Check::Found);
-        for i in 21..MAX_ENTRIES + 19 {
-            assert_eq!(slot_history.check(i), Check::NotFound, "found: {i}");
-        }
-        assert_eq!(slot_history.check(MAX_ENTRIES + 20), Check::Future);
-    }
-
-    #[test]
-    fn slot_history_test_same_index() {
-        solana_logger::setup();
-        let mut slot_history = SlotHistory::default();
-        info!("add 3,4");
-        slot_history.add(3);
-        slot_history.add(4);
-        assert_eq!(slot_history.check(1), Check::NotFound);
-        assert_eq!(slot_history.check(2), Check::NotFound);
-        assert_eq!(slot_history.check(3), Check::Found);
-        assert_eq!(slot_history.check(4), Check::Found);
-        slot_history.add(MAX_ENTRIES + 5);
-        assert_eq!(slot_history.check(5), Check::TooOld);
-        for i in 6..MAX_ENTRIES + 5 {
-            assert_eq!(slot_history.check(i), Check::NotFound, "i: {i}");
-        }
-        assert_eq!(slot_history.check(MAX_ENTRIES + 5), Check::Found);
-    }
-
-    #[test]
-    fn test_older_slot() {
-        let mut slot_history = SlotHistory::default();
-        slot_history.add(10);
-        slot_history.add(5);
-        assert_eq!(slot_history.check(0), Check::Found);
-        assert_eq!(slot_history.check(5), Check::Found);
-        // If we go backwards we reset?
-        assert_eq!(slot_history.check(10), Check::Future);
-        assert_eq!(slot_history.check(6), Check::Future);
-        assert_eq!(slot_history.check(11), Check::Future);
-    }
-
-    #[test]
-    fn test_oldest() {
-        let mut slot_history = SlotHistory::default();
-        assert_eq!(slot_history.oldest(), 0);
-        slot_history.add(10);
-        assert_eq!(slot_history.oldest(), 0);
-        slot_history.add(MAX_ENTRIES - 1);
-        assert_eq!(slot_history.oldest(), 0);
-        slot_history.add(MAX_ENTRIES);
-        assert_eq!(slot_history.oldest(), 1);
-    }
-}
+pub use {solana_clock::Slot, solana_slot_history::*};
```

Comment:
Should we add the deprecated message on this re-export?

Reply:
Done!